### PR TITLE
Database tooling

### PR DESCRIPTION
**Pull request by santiagosan93**
This PR focuses in implementing the [DataBase tooling update](https://www.notion.so/lewagon/Database-tooling-3974fc1465ac4014aa76197eabf87d95)

## Changes
- [x] Adds sqlite package installation instructions

### CODE REVIEW

File: _partials/windows_sqlite.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## SQLite
```
Comment (dmilon) [inline _partials/windows_sqlite.md:1]: When our the content is identical between Windows and Ubuntu partials, no need to create two. Our convention is to create a unique partial named `sqlite.md` which is going to hold the shared content between multiple OS.

File: _partials/ubuntu_sqlite.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## SQLite
+
+[SQLite](https://sqlite.org/index.html) is a database engine used to execute CRUD commands on databse files. Let's install it with `homebrew` .
```
Comment (dmilon) [inline _partials/ubuntu_sqlite.md:3]: If we put SQLite before PostgreSQL, then we can move the database intro sentence here and adapt it a bit: 

```suggestion
In a few weeks, we'll talk about databases and SQL. [SQLite](https://sqlite.org/index.html) is a database engine used to execute CRUD commands on database files. Let's install it:
```

Then we could update the PostgreSQL section intro sentence with something like: 
> Sometimes, SQLite is not enough and you will need a more advanced too called PostgreSQL, an open-source, robust and production-ready database system.

File: _partials/ubuntu_sqlite.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+Then run the following command:
+
+```bash
+sqlite3 -version
+```
+
+
+You should see a version. If not, ask a teacher.
```
Comment (dmilon) [inline _partials/ubuntu_sqlite.md:18]: Let's add our conventional emojis here: 

```suggestion
:heavy_check_mark:  If you see a version, you're good +1
:x:  If not, ask for a teacher
```

File: _partials/windows_sqlite.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+[SQLite](https://sqlite.org/index.html) is a database engine used to execute CRUD commands on databse files. Let's install it with `homebrew`.
```
Comment (dmilon) [inline _partials/windows_sqlite.md:3]: Let's copy-paste the new intro sentence here.

File: _partials/windows_sqlite.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+```bash
+sudo apt-get install sqlite3 libsqlite3-dev
+```
+
+Then run the following command:
+
+```bash
+sqlite3 -version
+```
+
```
Comment (dmilon) [inline _partials/windows_sqlite.md:16]: Extra empty line: 

```suggestion
```

File: _partials/windows_sqlite.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+You should see a version. If not, ask a teacher.
```
Comment (dmilon) [inline _partials/windows_sqlite.md:18]: Let's copy-paste end-of-section emojied sentences here :pray:

**Review by dmilon — CHANGES_REQUESTED** [review]
Nice work @santiagosan93 :raised_hands: Few comments below :pray:

**Comment by santiagosan93** [comment]
Thanks for the review @dmilon. Feedback has been implemented 👍

File: build.rb [hunk 7]
```diff
@@ -24,6 +23,8 @@
   ruby
   nvm
   yarn
+  macos_sqlite
+  vscode_extensions
```
Comment (dmilon) [inline build.rb:27]: I'ld rather keep the `vscode_extensions` partial just after the VS Code installation instructions partial, where it was. Do we need to have SQLite already installed when we install the VS Code SQLit extension? :thinking:

File: build.rb [hunk 8]
```diff
@@ -55,6 +55,8 @@
   ruby
   nvm
   yarn
+  sqlite
```
Comment (dmilon) [inline build.rb:58]: Same.

File: build.rb [hunk 9]
```diff
@@ -81,6 +82,8 @@
...
```
Comment (dmilon) [inline build.rb:85]: Same.

**Review by dmilon — COMMENTED** [review]
Thank you for the update @santiagosan93 :pray: Just a question about the dependency of VS Code SQLite extension on SQLITE :point_down:

File: build.rb (hunk 7 above)
Comment (dmilon) [reply build.rb:27]: I've uninstalled sqlite and the extension on my Linux machine and I can install back the extension without having sqlite binay installed yet. It's not working of course but the extension installation process is still working. Let's restore the `vscode_extensions` position then :pray:

**Pull request merged**